src/guards.js

```diff
@@ -105,4 +105,22 @@ export const isAskedAboutUsernameDiamond = (text) => {
  */
 export const isAskedWhoMadeMe = (text) => {
     return /who(?:\s+(?:are|is) your)?\s+(?:made|created|owns|develop(?:s|ed|ers?)|maintain(?:s|ers?))(\s+you)?/.test(text);
+};
+
+/**
+ * @summary checks if the message asked for candidate score of another user
+ * @param {string} text
+ * @returns {boolean}
+ */
+export const isAskedForOtherScore = (text) => {
+    return /what(?: i|')s(?: the)? candidate score (?:for|of) \d+(?:$|\?)/.test(text);
+};
+
+/**
+ * @summary checks if the message asked for candidate score calculation formula
+ * @param {string} text
+ * @returns {boolean}
+ */
+export const isAskedForScoreFormula = (text) => {
+    return /(?:what|how) (?:is )?candidate score(?:(?: is)? calculated| formula)?(?:$|\?)/.test(text);
```

Comment:
The thing with regex, is that there are fewer possible permutations, which may be a good or bad thing.

`what is the formula for the candidate score`
`how to calculate the candidate score`
etc..

Reply:
Agreed - I wanted to be conservative here about what it matches as the previous iteration of the guards (non-regex) were a bit too permissive in terms of what is matched, resulting in some false positives. Now that it is covered by tests, we can likely expand the expression to match the us cases you listed